docs/framework/configure-apps/file-schema/runtime/gccpugroup-element.md

```diff
@@ -69,5 +69,5 @@ Specifies whether garbage collection supports multiple CPU groups.
 ## See also
 - [Runtime Settings Schema](../../../../../docs/framework/configure-apps/file-schema/runtime/index.md)
 - [Configuration File Schema](../../../../../docs/framework/configure-apps/file-schema/index.md)
-- [How to: Disable Concurrent Garbage Collection](https://msdn.microsoft.com/library/ba2c6c67-5778-497c-9fac-5f793b5500c7)
+- [How to: Disable Concurrent Garbage Collection](/previous-versions/dotnet/netframework-4.0/at1stbec(v=vs.100))
```

Comment:
I've created a new PR #10247 that (if approved) would allow us to link to this as follows:

```suggestion
- [To disable concurrent garbage collection](gcconcurrent-element.md#to-disable-background-garbage-collection)
```